auth0/src/main/java/com/auth0/android/authentication/AuthenticationAPIClient.kt

```diff
@@ -285,14 +282,16 @@ public class AuthenticationAPIClient @VisibleForTesting(otherwise = VisibleForTe
      * @param password   of the user and must be non null
      * @param username   of the user and must be non null
      * @param connection of the database to create the user on
+     * @param userMetadata to set upon creation of the user
      * @return a request to start
      */
     @JvmOverloads
     public fun createUser(
         email: String,
         password: String,
         username: String? = null,
-        connection: String
+        connection: String,
+        userMetadata: Map<String, String>? = null
```

Comment:
optional parameter with a default value of `null`. The `JvmOverloads` annotation covers the java use case by generating one extra method with the added parameter